README.rst

```diff
@@ -1,130 +1,117 @@
-You might also want to visit `our website <http://coala-analyzer.org/>`_.
+.. image:: https://cloud.githubusercontent.com/assets/7521600/15992701/ef245fd4-30ef-11e6-992d-275c5ca7c3a0.jpg
+
+coala: Language Independent Code Analysis
+-------------
+coala is a **language agnostic code analysis toolkit**. With coala, users
+can create rules and standards to which the code should conform to. coala
+has an **user-friendly interface** that is completely customizable. It can
+be used in any environment and is completely modularized.
+
+coala has a set of official bears (plugins) for 37 languages including
+**C/C++**, **Python**, **JavaScript**, **CSS** and many more in addition
```

Comment:
maybe rephrase: ``including popular languages, such as: x, y, z, and many more, in addition to some generic language-independent algorithms.``